src/grid/initial_conditions.cpp

```diff
@@ -508,9 +508,9 @@ void Grid3D::Square_Wave(parameters const &P)
  *  \brief Initialize the grid with a Riemann problem. */
 void Grid3D::Riemann(parameters const &P)
 {
-  size_t const istart = H.n_ghost;
-  size_t const iend   = H.nx - H.n_ghost;
-  size_t jstart, kstart, jend, kend;
+  int const istart = H.n_ghost;
```

Comment:
Why are all these `int`? If the issue is with `size_t` underflowing then we should figure out different logic since we decided to move to using `size_t` for indexing.

Reply:
Fixed in [a2e1844](https://github.com/cholla-hydro/cholla/pull/315/commits/a2e18445686171c368dfa9113bf03476d5d90cbf)